.github/PULL_REQUEST_TEMPLATE.md

```
@@ -1,34 +1,26 @@
-<br>
+### ⚓ Related Issue
+- #이슈번호
 
 ### 🥥 Contents
-<!-- 이 PR에서 작업한 내용에 대해 알려주세요! -->
-
-이런 작업을 했습니다
-
-<br>
+<!-- 작업한 주요 내용 요약 -->
+- 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)